src/raul/card

```diff
@@ -0,0 +1,67 @@
+public class Main {
+    public static void main(String[] args){
+
+        String [] numbertCards = {"4111111111111111", "4111111111111", "4012888888881881",
+                "378282246310005", "6011111111111117", "5105105105105100", "5105105105105106",
+                "9111111111111111"};
+        for (String vendor: numbertCards) {
+            printWhoVendor(vendor, getIssuer(vendor));
+        }
+
+    }
+    private static String getIssuer(String cardNumber){
+
+        String namecard = "Unknow";
+
+        int contNumber = cardNumber.length();
+
+        if(contNumber == 13){
+            namecard = isCardVisa(cardNumber) ? "Visa" : "Unknow";
+
+        }else {
+            if (contNumber == 15) {
+                namecard = isCardAmex(cardNumber) ? "Amex" : "Unknow";
+            }else{
+                if (contNumber == 16){
+                    if(isCardVisa(cardNumber)){
+                        namecard =  "Visa";
+                    }else{
+                        if(isCardDiscover(cardNumber)){
+                            namecard =  "Discover";
+                        }else{
+                            if(isCardMastercard(cardNumber)){
+                                namecard =  "Mastercard";
+                            }
+                        }
+                    }
+
+                }
+            }
+
+        }
+
+        return namecard;
+    }
+
+    private static boolean isCardVisa(String stringNum){
+        int digfirst = Integer.parseInt(stringNum.substring(0,1));
```

Comment:
you can use `charAt(0)` here instead of `substring(0,1)`. Also, it might not be necessary to convert the value to `int`, chars returned could be compared instead.